src/rules/ec2/EC2SecurityGroupOnlyTcp443.ts

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { parse } from 'path';
+import { CfnResource, Stack } from 'aws-cdk-lib';
+import { CfnSecurityGroupIngress, CfnSecurityGroup } from 'aws-cdk-lib/aws-ec2';
+import { NagRuleCompliance, NagRules } from '../../nag-rules';
+
+/**
+ * Security Groups should only allow inbound access to tcp443
+ * @param node the CfnResource to check
+ */
+export default Object.defineProperty(
+  (node: CfnResource): NagRuleCompliance => {
+    if (node instanceof CfnSecurityGroup) {
+      const ingressRules = Stack.of(node).resolve(node.securityGroupIngress);
+
+      if (ingressRules != undefined) {
+        //For each ingress rule, check that only TCP 443 is allowed from 0/0
```

Comment:
Incline comments describing the `for loops` and `if statements` need to be removed

Reply:
resolved as suggested.